src/rng.js

```diff
@@ -1,7 +1,14 @@
 import crypto from 'crypto';
 
-const rnds8 = new Uint8Array(16);
+const poolSize = 4;
+const rnds8Pool = new Uint8Array(16 * poolSize);
+let poolPtr = -1;
```

Comment:
I'm fine with the `let poolPtr = rnds8Pool.length;` change, but I'd like to keep `poolSize` constant, as the code will be more readable and less error prone in terms of future changes this way. As for the performance, it won't make any difference. WDYT?

Reply:
As for the `64` -> `256` pool size increase, it will mean slightly higher memory footprint (+ 240 bytes when compared with the current `master`). But probably it should be a good trade-off for the users.

Reply:
Partially applied the change in b73e4ab7da508afe5942101c601ffad61253a0ec